Dockerfile

```diff
@@ -17,7 +18,26 @@ RUN buildDeps='libmagic-dev' && \
     rm -rf /var/lib/apt/lists/* && \
     bundle install --deployment --without development test && \
     apt-get purge -y --auto-remove $buildDeps && \
-    whenever --update-crontab
+    mkdir -p log && \
+    ln -sfn /dev/stdout log/production.log && \
+    bundle exec whenever --update-crontab && \
+    { echo BUNDLE_APP_CONFIG=$BUNDLE_APP_CONFIG; crontab -l; } | crontab -
```

Comment:
Thanks. Wouldn't this better be placed in something like `/etc/profile.d` or so?